modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/registries/RegistryLookup.scala

```diff
@@ -14,25 +14,26 @@ package com.snowplowanalytics.iglu.client.resolver
 package registries
 
 // Java
+import cats.effect.Sync
+
 import java.net.UnknownHostException
 
 // Scala
 import scala.util.control.NonFatal
 
 // cats
-import cats.{Eval, Id}
 import cats.data.{EitherT, OptionT}
-import cats.effect.Sync
-import cats.implicits._
 import cats.effect.implicits._
+import cats.implicits._
+import cats.{Eval, Id}
```

Comment:
Can we add a commit to remove the `Eval` instance? It's been a mistake and we can fix it in this breaking release.